### PR TITLE
Update insider-risk-management-settings.md

### PR DESCRIPTION
**Pull request by msbemba**
Priority user groups is no longer in preview, updated the document accordingly.

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9670

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/insider-risk-management-settings.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Insider risk management settings apply to all insider risk management policies,
 - [Policy timeframes](#policy-timeframes)
 - [Intelligent detections](#intelligent-detections)
 - [Export alerts](#export-alerts)
-- [Priority user groups (preview)](#priority-user-groups-preview)
+- [Priority user groups](#priority-user-groups-preview)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/insider-risk-management-settings.md]:
```suggestion
- [Priority user groups](#priority-user-groups)
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@msbemba validation warning, please review. Thanks

**Comment by yogkumgit** [comment]
@msbemba Gentle reminder

**Comment by msbemba** [comment]
> @msbemba Gentle reminder

This has been fixed now.

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by robmazz** [comment]
This was updated in a separate PR.

**Pull request closed without merging**